packages/plugins/plugin-map/src/meta.ts

```diff
@@ -9,8 +9,9 @@ export const MAP_PLUGIN = 'dxos.org/plugin/map';
 export const meta = {
   id: MAP_PLUGIN,
   name: 'Maps',
-  description: 'Display objects on maps.',
+  description: `Activate Maps to plot data from your tables on an interactive globe. Just add Lat/Long fields to your schema to see them on a map. You can also plot points on the map while chatting with your AI assistant.`,
```

Comment:
interactive map or globe.
i think this is an example where we would really benefit from 2 screenshots at least, but for later.